src/sinks/file/mod.rs

```diff
@@ -96,6 +96,9 @@ pub enum Compression {
     /// Gzip compression.
     Gzip,
 
+    /// Zstandard compression.
```

Comment:
The new compression algorithm should be added to `website/cue/reference/components/sinks/file.cue`

Reply:
done in the thread below, but we have question about levels